{{ cookiecutter.package_name|slugify }}/pyproject.toml

```diff
@@ -32,6 +32,7 @@ version_files = ["pyproject.toml:version"]
 fastapi = "^0.70.1"
 gunicorn = "^20.1.0"
 {%- endif %}
+poethepoet = "^0.12.2"
```

Comment:
For non-deployable projects (i.e. projects that are Python packages), I'd prefer not to make `poethepoet` a required dependency. To solve that, could you wrap this line in:
```yaml
{%- if cookiecutter.with_fastapi_api|int or cookiecutter.with_streamlit_app|int %}
poethepoet = "^0.12.2"
{%- endif %}
```
and then also add it as a conditional development dependency with:
```yaml
{%- if not cookiecutter.with_fastapi_api|int and not cookiecutter.with_streamlit_app|int %}
poethepoet = "^0.12.2"
{%- endif %}
```

Reply:
Good idea. Done. Done :)